src/models/room.js

```diff
@@ -1,11 +1,27 @@
 "use strict";
+var mongoose = require("mongoose");
 
-class Room {
-	static findAll(q) {
-		return Promise.resolve([]);
+var roomschema = mongoose.Schema({
+	room_id: {
```

Comment:
Mongoose automatically gives models a `_id` field. So we can remove this `room_id` field.